src/lib/token_list_gen.ts

```diff
@@ -111,24 +122,29 @@ export const generateTokenList = async (
 
   console.log(`List has ${l2TokenList.length} bridged tokens`);
 
-  if(options && options.includeUnbridgedL1Tokens){
-    const l1AddressesOfBridgedTokens = new Set(tokens.map((token)=> token.l1TokenAddr.toLowerCase()))
-    const unbridgedL1Tokens:ArbTokenInfo[] = l1TokenList.tokens.filter((l1TokenInfo)=>{
-      return !l1AddressesOfBridgedTokens.has(l1TokenInfo.address.toLowerCase()) && l1TokenInfo.chainId === +l1Network.chainID
-    }).map((l1TokenInfo)=>{
+  const allOtherTokens = l1TokenList.tokens.filter(
+    (l1TokenInfo) => l1TokenInfo.chainId !== parseInt(l2Network.chainID)
+  ).map((l1TokenInfo)=>{
       return {
         chainId: +l1TokenInfo.chainId,
         name: l1TokenInfo.name,
         address: l1TokenInfo.address,
         symbol: l1TokenInfo.symbol,
         decimals: l1TokenInfo.decimals,
         logoURI: l1TokenInfo.logoURI
-
       }
+  })
+
+  if(options?.includeAllL1Tokens) {
+    l2TokenList = l2TokenList.concat(allOtherTokens)
```

Comment:
naming of "l2TokenLost" is slightly awkward now, but idk maybe its fine

Reply:
renamed to `arbifiedTokenList`